modules/5_clustering_umap/clustering_umap.qmd

```diff
@@ -428,6 +429,34 @@ for(ident in names(plist)) {
 cat(":::\n")
 ```
 
+## Clustering Tree
```

Comment:
Add level-3 header for Clustering Resolution (above), and Clustering Tree (here), below level-2 header for Clustering